index.html

```diff
@@ -7,14 +7,202 @@
+                    <h2>Group Therapy</h2>
+                    <p>
+                        I'm a paragraph. Click here to add your own text and edit me. It's easy. Just click "Edit Text" or double click me to add your own content and make changes to the font.
+                    </p>
+                </section>
+            </div>
+            <div class = "col-md-4 g-0">
+                <section class = "couples">
+                    <h3>03.</h3>
+                    <h2>Couples Therapy</h2>
+                    <p>
+                        I'm a paragraph. Click here to add your own text and edit me. It's easy. Just click "Edit Text" or double click me to add your own content and make changes to the font.
+                    </p>
+                </section>
+            </div>
+        </div>
+        <div class = "row">
+            <div class = "col">
+                <button class = "buttons" id = "services">View More Services</button>
+                <a href = "#">&rarr;</a>
+            </div>
+        </div>
+        </section>
+    </div>
+    <!--Articles Section-->
+    <div class = "container g-0">
+        <div class = "row text-center">
+            <section class = "articles">
+                <div class = "col">
+                        <h3 class = "psych-articles">Articles about Psychology</h3>
+                </div>
+                <div class="card-group">
+                    <div class="card">
+                    <div class="card-body text-center">
+                        <a href = "#">
+                            <img class = "card-img-top img-fluid" src = "./img/flower-blue.png">
+                            <h3 class = "card-title">Understanding PTSD</h3>
+                        </a>
+                    </div>
+                    </div>
+                    <div class="card">
+                    <div class="card-body text-center">
+                        <a href = "#">
+                            <img class = "card-img-top img-fluid" src = "./img/article-argue.png">
+                            <h3 class = "card-title">Why it's okay to argue</h3>
+                        </a>
+                    </div>
+                    </div>
+                    <div class="card">
+                    <div class="card-body text-center">
+                        <a href = "#">
+                            <img class = "card-img-top img-fluid" src = "./img/article-de-stress.png">
+                            <h3 class = "card-title">10 Ways to de-stress</h3>
+                        </a>
+                    </div>
+                    </div>
+                </div>
+            <div class = "row">
+                <div class = "col">
+                    <button class = "buttons">
+                        More Articles
+                    </button>
+                </div>
+            </section>
+        </div>
+    </div>
+    <!--Social Media-->
+    <div class = "container">
+        <footer>
```

Comment:
Remember you don't have to apply Bootstrap classes to `div`s only - you don't need the div on line 181 here, you can just apply the `container` class to the `<footer>` directly